examples/sunpy_other_packages/reprojection_aia_euvi_mosaic.py

```diff
@@ -183,7 +184,8 @@
 # little.
 
 outmap = sunpy.map.Map((array, header))
-outmap.plot_settings = maps[0].plot_settings
+outmap.cmap = maps[0].cmap
```

Comment:
Pass these as kwargs to `plot` instead of setting them explicitly. To incorporate the altered `vmin`, you may need to do something like,

```python
norm = maps[0].norm
norm.vmin = 400
```

and then

```python
im = outmap.plot(cmap=maps[0].cmap, norm=norm)
```